src/notation/inotationinteraction.h

```diff
@@ -292,6 +292,8 @@ class INotationInteraction
     virtual async::Channel<ShowItemRequest> showItemRequested() const = 0;
 
     virtual void setGetViewRectFunc(const std::function<RectF()>& func) = 0;
+
+    virtual void setLogicClickPos(const PointF& logicPos) = 0;
```

Comment:
(Good job about understanding the stuff with these "interface" classes correctly!)